### PR TITLE
Updates supporting payloads as detached, and draft 07

### PR DESCRIPTION
**Pull request by SteveLasker**
## Summary

- [x] `Feed_id` becomes `subject`
- [ ] `Payload` becomes `statement`
- [x] Statements are only supported as detached payloads, where the hash is placed in the payload
- [x] The `content-type` is set to the content type of the detached payload type
- [ ] Rough proposal for storing statements on the DataTrails Attachments API, reducing the 
- [x] `location-hint` created as a DataTrails property on the protected header, 
- [ ] `arc_statement-location-hint` event attribute lifted from the DataTrails protected header property
- [x] `scitt_issuer` event attribute lifted from the `cwt-claims.sub` protected header
- [x] `scitt_subject` event attribute lifted from the `cwt-claims.sub` protected header
- [ ] `scitt_statement-payload` lifted from the protected header statement property (although I am thinking we may want to leave the payload `null`, and create a `datatrails_statement-hash` protected header label
- [ ] `scitt_content-type` event attribute lifted from the protected header

### CODE REVIEW

File: content/developers/developer-patterns/scitt-api/index.md [hunk 1]
```diff
@@ -89,49 +96,82 @@ For the Quickstart, create a testing [COSE Key](https://cose-wg.github.io/cose-s
...
+    cat > $STATEMENT_FILE <<EOF
+    {
+        "build-server": "22",
+        "test-results": "passed",
+        "compliance-22": "compliant"
+    }
+    EOF
+    ```
+
+1. Optionally save the statement in the [DataTrails Events Attachments API](/developers/api-reference/events-api/#adding-attachments)
```
Comment (honourfish) [inline content/developers/developer-patterns/scitt-api/index.md:113]: I thought we were going to punt the attachments api?

I think the minimum the sample script does is take a file, hash it and add that to the payload, while any file metadata is stored in the protected header?

Comment (robinbryce) [reply content/developers/developer-patterns/scitt-api/index.md:113]: I think this is  ok though ? We are just saying, "we have an attachments api, and you can use it with scitt like this if you want" ?

Comment (honourfish) [reply content/developers/developer-patterns/scitt-api/index.md:113]: I think this is probably going to be the first touch experience for alot of developers that want to try/use scitt with datatrails.

I think it maybe better to have a separate how-to for detached statements and how to deal with them, using the python sample scripts (that will be written). Then that how-to can go into more detail about storage (if we want) but talking about what goes into the payload and what goes in the protected headers...etc

I think its very good having a statement saying something along the lines of `if your statement is more complex than a small amount of json, see this how-to ... (insert link)`

Comment (SteveLasker) [reply content/developers/developer-patterns/scitt-api/index.md]: latest change reflects us keeping the attached and detached scenarios. I've kept as one doc for now to ease our viewing. 
I can split into separate docs if needed as it becomes functional.

**Comment by SteveLasker** [comment]
Pushed latest changes from our meeting today.

**Comment by SteveLasker** [comment]
Fixed with #863, #870

**Pull request closed without merging**